modules/apps/frontend-theme/frontend-theme-dialect/src/css/dialect/variables/_buttons.scss

```diff
@@ -123,3 +125,59 @@ $btn-link: (
 		color: var(--color-action-primary-disabled),
 	),
 );
+
+$btn-danger: 
```

Comment:
There is a space after the `:` for all the variables. We should remove this white-space.